### DISCUSSION TITLE
Specific Observations in Generated Data

### DISCUSSION THREAD

**Discussion by swhasans** [synthetichealth/synthea#1529] [Q&A] [answered]
I need to make sure that every patient generated has specific observations (Creatinine, FBS, GTT, HbA1c, eGFR, UACR, Blood Pressure, Total Cholesterol, LDL, HDL, Triglycerides, and Vitamin B12).

I'm modifying the 'Metabolic Syndrome Standards of Care' module to include states that generate these observations. Despite this, the observations are not appearing in the output as expected.

When I run the command:
`java -jar synthea-with-dependencies.jar -p 100 -m metabolic_syndrome_care`

the 'Observations' column is not seen. However, when I run:
`java -jar synthea-with-dependencies.jar -p 100`

some observations are listed, but not all the required ones.

How to ensure all specified results are consistently generated for each patient? Any suggestions on correctly modifying the module or additional steps to take would be greatly appreciated.

**Comment by jawalonoski** [comment, marked as answer]
https://github.com/synthetichealth/synthea/wiki/The--M-Feature

With the second command, if you downloaded that jar file, it includes the original version of the module: hence not the observations you added. If you built the jar from source, you should be fine as long as you overwrote the original module file.

So, you either need to rebuild the jar to include your changes, or use the `-d localModulesDirPath` option to _add_ a local set of modules to the defaults, but that _may not replace_ the original metabolic syndrome care (if so, both would be running in parallel).

**Reply by swhasans** [reply]
Thanks for your help! There were two issues: first as you said, I wasn't using the -d flag, and second, I didn't fill in all the details while creating the states in the 'State Editor'. After correcting these and creating a dir for the "custom" module, I used `./run_synthea -d ./custom_module/`, and it worked fine. All the observations are generated correctly.
